opentelemetry-operator/chart/values.yaml

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 opentelemetry-operator:
+  crds:
+    create: false
```

Comment:
This is disabled, because we're deploying the CRDs dedicatedly?

Why don't we use the operator functionality here and drop our manual adding of CRDs?

Reply:
Yes, that is the reason.

I understood that it is [best practice](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you) to store them in a /crds folder rather than in the /template folder